src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -508,7 +511,16 @@ public boolean requiresReads() {
     }
 
     /**
-     * Does this tool require intervals? Traversals types and/or tools that do should override to return true.
+     * Should this tool use read caching to optimize forward-only queries on coordinate sorted read inputs ?
+     *
+     * @return true if read caching should be used, otherwise false
+     */
+    public boolean useReadCaching() {
```

Comment:
This could be a `getReadsLookaheadWindowSize` instead, returning a value with the meaning of disable (0? -1?) and let each walker-class to choose how the cache is handled. A tool can always implement an argument to modify this value.